dev/run-tests.py

```diff
@@ -594,7 +651,18 @@ def main():
 
     modules_with_python_tests = [m for m in test_modules if m.python_test_goals]
     if modules_with_python_tests:
-        run_python_tests(modules_with_python_tests, opts.parallelism)
+        # We only run PySpark tests with coverage report in one specific job with
+        # Spark master with SBT in Jenkins.
+        is_sbt_master_job = (
+            os.environ.get("AMPLAB_JENKINS_BUILD_PROFILE", "") == "hadoop2.7"
```

Comment:
This environment variables were checked before at https://github.com/apache/spark/pull/17669

Reply:
should we hardcode this here or as a new env var to enable this explicitly?

Reply:
Yea.. that's actually one of the next steps :D.